src/components/skills-quiz/SkillsQuizStepper.jsx

```diff
@@ -334,7 +376,13 @@ const SkillsQuizStepper = ({ isStyleAutoSuggest }) => {
               Go back
             </Button>
             <Stepper.ActionRow.Spacer />
-            <Button onClick={() => setCurrentStep(STEP3)}>Continue</Button>
+            <Button data-testid="skills-continue-button" onClick={() => setCurrentStep(STEP3)}>
+              <FormattedMessage
+                id="enterprise.skills.quiz.v1.continue.button.label"
+                defaultMessage="Continue"
+                description="Continue button label on the skills quiz v1 page."
```

Comment:
What will happen on continue? Can we add little bit more context in the description?